src/routes/living-manage-offer/components/ConfirmOfferManager.js

```diff
@@ -2,20 +2,34 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Caption } from '@integreat-app/shared'
 import Spinner from 'react-spinkit'
+import { GONE, NOT_FOUND } from 'http-status-codes'
 
 export class ConfirmOfferManager extends React.Component {
   static propTypes = {
     send: PropTypes.func.isRequired,
     sending: PropTypes.bool.isRequired,
     success: PropTypes.bool.isRequired,
-    serverError: PropTypes.string
+    serverError: PropTypes.shape({
+      status: PropTypes.number,
+      message: PropTypes.string
+    })
   }
 
   componentWillMount () {
     // Immediately send confirmRequest
     this.props.send('POST', `/confirm`)
   }
 
+  getErrorMessage () {
+    switch (this.props.serverError.status) {
+      case GONE:
+        return 'Der Bestätigungslink ist ungütltig.'
```

Comment:
Why don't you use the servers error messages here? I mean we already have them :)